src/index.html

```diff
@@ -9,6 +9,82 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <div class="logo">
+        <a href="#home">
+          <img 
+            src="images/logo.png" 
+            alt="moyoLogo"
+          >
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav_links">
+          <li class="list">
+            <a 
+              href="#apple"
+              class="is-active technic">Apple
+            </a>
```

Comment:
```suggestion
      <nav class="nav">
        <ul class="nav_list">
          <li class="nav_item">
            <a 
              href="#apple"
              class="nav_link is-active">Apple
            </a>
```
It is more informative, fix it everywhere